conf/schemas/cloudtrail.json

```diff
@@ -75,5 +75,23 @@
         "vpcEndpointId"
       ]
     }
+  },
+  "cloudtrail:insight": {
+    "schema": {
+      "eventVersion": "string",
+      "eventTime": "string",
+      "awsRegion": "string",
+      "eventID": "string",
+      "eventType": "string",
+      "recipientAccountId": "string",
+      "sharedEventID": "string",
+      "eventCategory": "string",
+      "insightDetails": {}
+    },
+    "parser": "json",
+    "configuration": {
+      "json_path": "Records[*]",
+      "optional_top_level_keys": []
```

Comment:
We can remove this entry since it is empty.

Reply:
Thanks done.